include/api.php

```diff
@@ -4498,6 +4498,11 @@ function api_account_update_profile($type)
 	}
 
 	Worker::add(PRIORITY_LOW, 'ProfileUpdate', api_user());
+	// Update global directory in background
+	$url = $_SESSION['my_url'];
```

Comment:
I'm not sure, if the session variable is defined here, when working over the API.

Reply:
I tested it and it works, even with a totally new session.

Reply:
It is set in the various authentication methods:
- Classic: https://github.com/friendica/friendica/blob/3f448d9a422ab8bc886ee760ab8f1e1cddad099a/include/security.php#L57
- OAuth: https://github.com/friendica/friendica/blob/82df294e31a938ede7b5374d53094c1b7034df29/src/Network/FKOAuth1.php#L55
- Magic auth: https://github.com/friendica/friendica/blob/6b622fa9e4c7c62c874c2e2c2b73db2a0a1274d4/index.php#L127

Reply:
I would prefer using the url from the contact table.